src/index.js

```diff
@@ -115,7 +138,11 @@ const getLikedPosts = (timestamp) => {
           for (let i = 0; i < likedPosts.length; i++) {
             const currentPost = likedPosts[i];
             if (Array.isArray(currentPost.photos)) {
-              currentPost.photos.forEach(pushToArray);
+              const filebase = cleanFileName(`${currentPost.blog_name}_${currentPost.timestamp}`);
```

Comment:
Thinking about it more, timestamp by itself would not necessarily be unique, so that's probably a bad idea.
How I about I just set it up as:

"If you want notes, then the files are named to {blogname}_{timestamp}{image number if applicable}"

that only adds one param then. Keeps things simpler.